benchmark/largesetresult/Makefile

```diff
@@ -1,6 +1,8 @@
 ## Setup
+SHELL := /bin/bash
+SRC = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
+
 setup:
-	go get github.com/Masterminds/glide
```

Comment:
can you get the latest `dep` here? It seems dep cannot download itself.

All of Travis test have this
```
dep ensure
dep: WARNING: Unknown field in manifest: prune
```

Reply:
Absolutely